proto/prysm/v1alpha1/beacon_block.proto

```diff
@@ -436,6 +441,82 @@ message BlindedBeaconBlockBodyBellatrix {
     ethereum.engine.v1.ExecutionPayloadHeader execution_payload_header = 10;
 }
 
+message SignedBeaconBlockDenebAndBlobs {
+    SignedBeaconBlockDeneb block = 1;
+    repeated SignedBlobSidecar blobs = 2 [(ethereum.eth.ext.ssz_max) = "4"];
+}
+
+message SignedBeaconBlockDeneb {
+    // The unsigned beacon block itself.
+    BeaconBlockDeneb block = 1;
+
+    // 96 byte BLS signature from the validator that produced this block.
+    bytes signature = 2 [(ethereum.eth.ext.ssz_size) = "96"];
+}
+
+message BeaconBlockDenebAndBlobs {
+    BeaconBlockDeneb block = 1;
+    repeated BlobSidecar blobs = 2 [(ethereum.eth.ext.ssz_max) = "4"];
+}
+
+message BeaconBlockDeneb {
```

Comment:
Same here, can't we have a single `BeaconBlock` proto that takes different bodies?